suites/desktop/CMakeLists.txt

```diff
@@ -100,6 +100,17 @@ if(NOT json_POPULATED)
   add_subdirectory(${json_SOURCE_DIR} ${json_BINARY_DIR})
 endif()
 
+FetchContent_Declare(
+  cxxopts
+  GIT_REPOSITORY https://github.com/jarro2783/cxxopts.git
+  GIT_TAG        v2.2.0
+)
+FetchContent_GetProperties(cxxopts)
```

Comment:
Do we need to use both `cxxopts` and `dimcli`?

Reply:
Currently, Marketa is using cxxopts. What I suggest is to fetch them inside Marketa's CMake.

Reply:
I agree (and don't have a preference), I just wanted to point that out since it way moved to the top level CMakeLists in this branch.